client/src/pages/CalendarPage/CalendarPage.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import "react-calendar/dist/Calendar.css";
+import "./CalendarOverride.css";
+import Calendar from "react-calendar";
+import styles from "./CalendarPage.module.css";
+import TaskCard from "../../components/TaskCard/TaskCard.js";
+import CalendarHeader from "../../components/CalendarHeader/CalendarHeader.js";
+
+const dayLabelFormatter = (locale, label) => {
+  return label.toString().slice(0, 1);
+};
+
+const dailyTaskDateFormatter = (date) => {
+  const months = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+  ];
+  return String.raw`${
+    months[date.getMonth()]
+  } ${date.getDate()}, ${date.getFullYear()}`.toString();
+};
+
+const CalendarPage = () => {
+  const [currDate, onDateChange] = useState(new Date());
+  const children = [
+    {
```

Comment:
Although this is fine for now, I would suggest to put the `children` array in another file and folder under something such as `tmp/testData.js` and then have it exported. With this way, we'll be able to continue to use the testData but not have it cluttering the CalendarPage component.

It'll look something like the following

In `/tmp/testData.js`
```
export const testData = [
{     title: "GDSC Meeting",
      locationText: "DH2020",
      startTimeText: "05:00PM",
      endTimeText: "07:00PM",
      isOngoing: false,
}, ..........]
```

In `CalendarPage.js`
```
import testData from "/path/to/tmp/testData

const tasks = testData
const cards = tasks.map(() => {})
.....
```

Reply:
Additionally, `Text` can be removed and it can just be 
```

Reply:
I will take note of this for future dummy data. If we have not linked the backend and frontend by next PR, I will move the dummy data accordingly.

Reply:
Noted.